src/main/java/tool/Ui.java

```diff
@@ -0,0 +1,190 @@
+package tool;
+
+import java.util.Scanner;
+
+import task.Task;
+import tool.TaskList;
+
+/**
+ * Represents the tool to deal with user interactions.
+ */
+public class Ui {
+    private static final String DIVIDER = "        ------------------------------------------------------------";
+
+    private static final String POSSIBLE_COMMANDS = "        TODO     --- todo [task name]\n" +
+                                                    "        DEADLINE --- deadline [task name] /by [yyyy-mm-dd HH:MM]\n" +
+                                                    "        EVENT    --- event [task name] /from [yyyy-mm-dd HH:MM] /to [yyyy-mm-dd HH:MM]" ;
+
+    /**
+     * Constructs a Ui object.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Prints greetings to user.
+     */
+    public void greetUser() {
+        System.out.println(DIVIDER);
+        System.out.println("        Hello! I'm Chronos.");
+        System.out.println("        What can I do for you?");
```

Comment:
I noticed that you're writing out the spaces yourself in several parts of the code, perhaps using "\t" would be easier as it adds a tab space?